src/Components/CreateImageWizard/formComponents/ReviewStepTables.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+
+import { Panel, PanelMain } from '@patternfly/react-core';
+import {
+  TableComposable,
+  Tbody,
+  Td,
+  Th,
+  Thead,
+  Tr,
+} from '@patternfly/react-table';
+import PropTypes from 'prop-types';
+
+import { UNIT_GIB, UNIT_MIB } from '../../../constants';
+
+export const FSReviewTable = ({ ...props }) => {
+  return (
+    <Panel isScrollable>
+      <PanelMain maxHeight="30ch">
+        <TableComposable
+          aria-label="File system configuration table"
+          variant="compact"
+        >
+          <Thead>
+            <Tr>
+              <Th>Mount point</Th>
+              <Th>File system type</Th>
+              <Th>Minimum size</Th>
+            </Tr>
+          </Thead>
+          <Tbody data-testid="file-system-configuration-tbody-review">
+            {props.fsc.map((r, ri) => (
```

Comment:
Would it be easier for someone unfamiliar with this code to understand what we're mapping over if we used a more descriptive variable name? It isn't immediately obvious to me what `r` stands for...

Reply:
Updated :+1: